python/py-beancount/Portfile

```diff
@@ -4,18 +4,22 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-beancount
+categories-append   finance
 version             2.2.1
+revision            1
```

Comment:
Does the revision need to be bumped since the contents are not changing?

Reply:
No, it does not.

Reply:
Ah, I'd thought that removing the supported_archs line might require a revision bump. If not, happy to remove it.

Reply:
Removed it.